test/unit/query/builder.js

```diff
@@ -7478,6 +7478,36 @@ describe('QueryBuilder', function() {
     );
   });
 
+  it('#1982 - should allow query comments in querybuilder', function() {
+    testsql(
+      qb()
+        .from('testtable')
+        .comment('/* Added comment */'),
```

Comment:
Also multiple comments, error throwing and ignore on non supported databasese options should be tested

Reply:
And integration test for the supported databases